templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -26,6 +26,10 @@ if [[ "$TEST" = "docs" || "$TEST" = "publish" ]]; then
   make PULP_URL="http://pulp" html
   cd ..
 
+  # Validate OpenAPI schema
+  echo "Validating OpenAPI schema..."
+  cmd_prefix bash -c "django-admin spectacular --file pulp_schema.yml --validate"
```

Comment:
Can this be `pulpcore-manager ...` ?

Reply:
I tested locally and it works, I'm switching to `pulpcore-manager`